processor_function/shared/cnab_builder.py

```diff
@@ -113,9 +116,31 @@ def deploy_aci(self):
 
         credential = AzureIdentityCredentialAdapter()
         aci_client = ContainerInstanceManagementClient(credential, self._subscription_id)
+
+        service_bus = ServiceBus()
+        service_bus.send_status_update_message(self._id, "ACI Deployment started", "Deploying ACI container: " + self._container_group_name)
+
         result = aci_client.container_groups.create_or_update(self._resource_group_name, self._container_group_name,
                                                               group)
 
         while result.done() is False:
```

Comment:
consider writing while not result.done() which is more pythonic

Reply:
fixed